nextflow_schema.json

```diff
@@ -130,7 +130,7 @@
                 },
                 "bt_indices": {
                     "type": "array",
-                    "default": "null"
```

Comment:
@KevinMenden did the `nf-core schema build` command put this here?

Need to fix that behaviour if so..

Reply:
https://github.com/nf-core/tools/issues/1074